data/languages/fr60.sinc

```diff
@@ -990,7 +1014,8 @@ CC: "HI" is cc=0xF { local tmp:1 = ((C | Z) == 0); export tmp; }
 
 # INT #u8 D AC 3+3a ---- SSP-=4,PS --> (SSP),SSP-=4,PC+2 --> (SSP),(TBR+0x3FC-u8x4) --> PC
 :INT #u8 is op8=0x1f & u8 {
-     local ea:4 = inst_next;
+    local ea:4 = inst_next;
+    packProgramStatus();
```

Comment:
I think this can be removed for `INT`. `_interrupt_prelude(PS, ea)` is a kludge that gets fixed up with the utron script.

As it's already being replaced with a script, ensuring the PS register is accurate is less important IMO.

Reply:
Okay, I just did a quick search for `PS` and added either one of these macros.  Interrupt code is always gonna be/look terrible, so I'm fine removing this if you think thats better